src/Form/KalastaticSettingsForm.php

```diff
@@ -42,26 +42,20 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     $form = array(
       'description' => array(
-        '#markup' => '<p>' . t('Static site framework for prototyping and building out CMS-less websites. See @link for more details.', array('@link' => $github_link)) . '</p>',
+        '#markup' => '<p>' . t('Static site framework for building out prototypes and styleguides. See @link for more details.', array('@link' => $github_link)) . '</p>',
       ),
       'kalastatic_src_path_wrap' => array(
         '#type' => 'fieldset',
-        '#title' => t('Kalastatic Paths'),
+        '#title' => t('Kalastatic'),
         '#collapsible' => FALSE,
         '#collapsed' => FALSE,
         'kalastatic_src_path' => array(
-          '#type' => 'textfield',
-          '#title' => $this->t('Source Path'),
-          '#default_value' => $config->get('kalastatic_src_path'),
-          '#size' => 60,
-          '#description' => t("Provide the path to Kalastatic, relative to Drupal root. If no path is supplied then it will be assumed that Kalastatic is inside a folder called 'kalastatic' in the root of the currently enabled theme."),
+          '#prefix' => '<h3>' . $this->t('Source Path') . ':</h3>',
+          '#markup' => '<pre>' . $config->get('kalastatic_src_path') . '</pre>',
```

Comment:
Rather than loading from `$config`, can we load straight from the Yaml file? Since we are only setting the config variables on install, it won't pick up any changes unless we manually change them. Since the config values don't actually inform anything, I think we can do away with them.

Reply:
Good point.